### DISCUSSION TITLE
Some suggestions

### DISCUSSION THREAD

**Discussion by naasking** [photoprism/photoprism#4658] [Q&A] [answered]
I think file manager mode which just shows the folder structure would be very helpful, even if no previews for those files are available, if only to load/download pictures that are too large to index according to the current settings (or whatever other error was encountered). If a file is found in this file manager mode that isn't indexed, ideally photoprism could display the reason why right here. So add every file to the index, but just don't index any properties if it doesn't pass the filters. Or perhaps, if you're in the library looking at a path under originals, add a button for each folder to "show all files under this path that aren't indexed" and you can do a short scan to display this list, and possibly the reason each file wasn't indexed. This would be a far more straightforward method of troubleshooting than [the checklist](https://docs.photoprism.app/getting-started/troubleshooting/#missing-pictures).

I also think it would be useful if photoprism could indicate somewhere when a scan is in progress. I often can only tell when I hear the server fan spinning up, or the system becomes slow or unresponsive, or I trigger a scan manually and it eventually tells me a scan is already in progress and then gives me an error message. There's what looks like a progress bar under library/index where I can start a scan, but this progress bar doesn't seem to work when a scan is in progress (I'm using sqlite).

**Comment by omerdduran** [comment, marked as answer]
Hi. About showing indexing progress there is already an open issue for that #639. For the first suggestion about a file manager with additional indexing details, I recommend creating a new GitHub issue.

**Reply by naasking** [reply]
Ok, I was just posting to foster a discussion on whether this was a good idea, but I'll just create the issue.
